frontend/src/views/Collection/style.ts

```diff
@@ -5,7 +5,6 @@ export const ViewGrid = styled.div`
   display: grid;
   grid-template-columns: repeat(8, 1fr);
   width: 100%;
-  margin-top: 128px;
```

Comment:
I added this margin to reflect the mock, was this top margin replaced with something else?

Reply:
Ooh yes, you're right lol The mock does have that 128px margin 🤦 My bad! I'll add it back!

https://www.figma.com/file/g0L0CfVzRc8r6HQa4NH1wJ/Publisher-Collection?node-id=0%3A1

Reply:
Done!